bundle/camunda-saas-bundle/src/main/resources/application.properties

```diff
@@ -12,21 +12,21 @@ camunda.connector.webhook.enabled=true
 camunda.connector.secret-provider.discovery.enabled=false
 camunda.connector.secret-provider.environment.enabled=false
 camunda.connector.secret-provider.console.enabled=false
-camunda.client.zeebe.execution-threads=10
-camunda.client.zeebe.defaults.max-jobs-active=32
-camunda.client.zeebe.defaults.stream-enabled=true
+camunda.client.execution-threads=10
+camunda.client.worker.defaults.max-jobs-active=32
+camunda.client.worker.defaults.stream-enabled=true
 camunda.client.mode=saas
 # to be provided in the deployment:
-# camunda.client.zeebe.grpc-address
-# camunda.client.zeebe.rest-address
+# camunda.client.grpc-address
+# camunda.client.rest-address
 connectors.log.appender=stackdriver
 # For specifying the size of inbound connectors activity log
 camunda.connector.inbound.log.size=10
 # For specifying the default time-to-live of inbound connectors messages
 # Default is 1h and can be overridden by the connector configuration
 #camunda.connector.inbound.message.ttl=PT1H
 # Disabling the default Operate client, we are configuring our own
-camunda.client.operate.enabled=false
+operate.client.enabled=false
```

Comment:
@jonathanlukas is that property still correct for the latest SDK? Asking as this PR is meant for `main` where we only use the Camunda client.

Reply:
Gotcha. This is not required anymore.

Reply:
DOne